PYME/LMVis/displayPane.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, panel, glCanvas, visFr):
         self._pc_clim_change = False
 
         #Colourmap
-        cmapnames = colormaps.cm.cmapnames
```

Comment:
I assume that making `cmapnames` a property of the class is to avoid an additional `list()` cast in `OnCmapChange()`??

It's a very minor point, but in the spirit of moving towards more readable code, it would be preferable to either make it private, or just do the additional cast (which should be cheap) - it would be a mistake to infer that it was safe to access the `.cmapnames` property from other classes.

Reply:
Well, I was thinking of the (possibly very unlikely but possible) case that some code injects new colour maps into the list during the lifetime of the displayPane. If we then re-evaluate the cmapnames the index could get off by one or two, unless I got the logic wrong when I thought about this (this is an ordered dict I guess, so perhaps not really an issue). Perhaps exceedingly unlikely but probably hard to debug if it occurred? So a private method would be preferable possibly, i.e. `_cmapnames`?

Reply:
That seems like sound logic, making a private variable name the best choice.